src/components/board/PostWriteForm.tsx

```
@@ -0,0 +1,228 @@
+    return (
+      <RequireMember>
+        <main className="min-h-screen bg-white">
+          <div className="container-x-lg flex min-h-[60vh] flex-col items-center justify-center gap-3 text-center">
+            <h1 className="text-h1 text-gray-900">운영진만 작성할 수 있어요</h1>
+            <p className="text-base text-gray-600">
+              권한이 있는 계정으로 로그인해주세요.
+            </p>
+            <button
+              onClick={() => router.push(backPath)}
+              className="mt-2 inline-flex items-center gap-1 rounded-full border border-gray-200 px-5 py-2.5 text-sm text-gray-600 transition-colors hover:bg-gray-50"
+            >
```

Biome (2.5.0)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)